src/main/java/com/nutrons/FRamework/CompMode.java

```diff
@@ -1,4 +1,4 @@
-package com.nutrons.FRamework;
+package com.nutrons.framework;
```

Comment:
This one is a pun, so we were considering keeping it, that being said, I can see it breaking other people's style checkers.